test/vanilla/low-level/Expected/AcceptanceTests/ConstantsLowLevel/constantslowlevel/rest/contants/_request_builders.py

```diff
@@ -107,15 +107,15 @@ def build_put_no_model_as_string_no_required_one_value_no_default_request(
     See https://aka.ms/azsdk/python/protocol/quickstart for how to incorporate this request builder
     into your code flow.
 
-    :keyword input:  Default value is "value1".
+    :keyword input:  Default value is None.
```

Comment:
Can we also update the documentation to say the other value is `"value1"`? I think playing with code [here](https://github.com/Azure/autorest.python/blob/autorestv3/autorest/codegen/models/parameter.py#L119) is good

Reply:
updated